prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Flexbox - align-items</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    /* your styles here 👇 */
+    div {
+      display: flex;
+    }
+
+    .card-wrapper {
+      align-items: center;
+      justify-content: center;
+    }
+
+    #card-deck {
+      flex-basis: calc(100% / 3);
```

Comment:
`align-items` and `justify-content` only affect "flex containers", which in this case would be the `#card-deck`.